tests/framework/resource/test_manager.py

```diff
@@ -41,74 +41,62 @@ def producer(self, _simulant_data: SimulantData) -> None:
         pass
 
 
-@pytest.mark.parametrize("r_type", RESOURCE_TYPES, ids=lambda x: f"r_type_{x}")
-def test_resource_manager_get_resource_group(r_type: str, manager: ResourceManager) -> None:
-    component = ResourceProducer("base")
-    r_names = ["foo"]
-    r_producer = component.producer
-    r_dependencies: list[str | Resource] = []
+@pytest.mark.parametrize(
+    "resource_class, type_string",
+    [
+        (Pipeline, "value"),
+        (ValueSource, "value_source"),
+        (MissingValueSource, "missing_value_source"),
+        (ValueModifier, "value_modifier"),
+        (Column, "column"),
+        (NullResource, "null"),
+    ],
+    ids=lambda x: {x.__name__ if isinstance(x, type) else x},
+)
+def test_resource_manager_get_resource_group(
+    resource_class: type, type_string: str, manager: ResourceManager
+) -> None:
+    producer = ResourceProducer("base").producer
 
-    group = manager._get_resource_group(r_type, r_names, r_producer, r_dependencies)
+    group = manager._get_resource_group([resource_class("foo")], producer, [])
 
-    assert group.type == r_type
-    assert group.names == [f"{r_type}.foo"]
-    assert group.producer == component.producer
+    assert group.type == type_string
+    assert group.names == [f"{type_string}.foo"]
+    assert group.producer == producer
     assert not group.dependencies
 
 
 def test_resource_manager_get_resource_group_null(manager: ResourceManager) -> None:
-    component = ResourceProducer("base")
-    r_names: list[str] = []
-    r_producer = component.producer
-    r_dependencies: list[str | Resource] = []
+    producer = ResourceProducer("base").producer
 
-    group_1 = manager._get_resource_group("column", r_names, r_producer, r_dependencies)
-    group_2 = manager._get_resource_group("column", r_names, r_producer, r_dependencies)
+    group_1 = manager._get_resource_group([], producer, [])
+    group_2 = manager._get_resource_group([], producer, [])
 
-    assert group_1.type == NULL_RESOURCE_TYPE
-    assert group_1.names == [f"{NULL_RESOURCE_TYPE}.0"]
-    assert group_1.producer == component.producer
+    assert group_1.type == "null"
+    assert group_1.names == ["null.0"]
+    assert group_1.producer == producer
     assert not group_1.dependencies
 
-    assert group_2.type == NULL_RESOURCE_TYPE
-    assert group_2.names == [f"{NULL_RESOURCE_TYPE}.1"]
-    assert group_2.producer == component.producer
+    assert group_2.type == "null"
+    assert group_2.names == ["null.1"]
+    assert group_2.producer == producer
     assert not group_2.dependencies
 
 
-def test_resource_manager_add_resources_bad_type(manager: ResourceManager) -> None:
```

Comment:
Is this just not a relevant test anymore?

Reply:
Since there is no string "type" parameter to the method, they can't really provide a bad type.